console-subscriber/src/lib.rs

```diff
@@ -643,3 +643,18 @@ impl WakeOp {
         }
     }
 }
+
+#[track_caller]
+pub(crate) fn spawn_named<T>(
+    task: impl std::future::Future<Output = T> + Send + 'static,
+    _name: &str,
+) -> tokio::task::JoinHandle<T>
+where
+    T: Send + 'static,
+{
+    #[cfg(tokio_unstable)]
```

Comment:
It looks like the builder already asserts this is set, but this doesn't hurt.

Reply:
yeah...it's possible people are using the crate without the builder. We may want to remove the assertion from the builder, in the case where the console is used with non-tokio runtimes like Rayon...